CONTRIBUTING.md

```diff
@@ -98,15 +101,30 @@ $ git commit
 Writing good commit logs is important. A commit log should describe what
 changed and why. Follow these guidelines when writing one:
 
-1. The first line should be 50 characters or less and contain a short
-   description of the change. All words in the description should be in
-   lowercase with the exception of proper nouns, acronyms, and the ones that
-   refer to code, like function/variable names. The description should
-   be prefixed with the name of the changed subsystem and start with an
+#### Commit Message Guidelines
+
+1. The first line (the header) should:
```

Comment:
It's referred to as the header below. I was just conserving the language so that later references to 'the header' would be understandable.